lnbits/extensions/admin/templates/admin/_tab_theme.html

```diff
@@ -0,0 +1,117 @@
+<q-tab-panel name="theme">
+  <q-card-section class="q-pa-none">
+    <h6 class="q-my-none">UI Management</h6>
+    <br />
+    <div>
+      <div class="row q-col-gutter-md">
+        <div class="col-12 col-md-6">
+          <p>Site Title</p>
+          <q-input
+            filled
+            type="text"
+            v-model="formData.lnbits_site_title"
+            label="Site title"
+          ></q-input>
+          <br />
+        </div>
+        <div class="col-12 col-md-6">
+          <p>Site Tagline</p>
+          <q-input
+            filled
+            type="text"
+            v-model="formData.lnbits_site_tagline"
+            label="Site tagline"
+          ></q-input>
+          <br />
+        </div>
+      </div>
+      <div>
+        <p>Site Description</p>
+        <q-input
+          v-model="formData.lnbits_site_description"
+          filled
+          type="textarea"
+          hint="Use plain text or raw HTML"
+        />
+      </div>
+      <br />
+      <div class="row q-col-gutter-md">
+        <div class="col-12 col-md-6">
+          <p>Default Wallet Name</p>
+          <q-input
+            filled
+            type="text"
+            v-model="formData.lnbits_default_wallet_name"
+            label="LNbits wallet"
+          ></q-input>
+          <br />
+        </div>
+        <div class="col-12 col-md-6">
+          <p>Denomination</p>
+          <q-input
```

Comment:
- should be drop-down, otherwise the user can type anything:
![image](https://user-images.githubusercontent.com/2951406/205637442-ca013d69-6ea3-4c40-a076-9c05ade3f585.png)

Reply:
User should be able to type anything.